src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.html.twig

```diff
@@ -1,8 +1,8 @@
 <!-- START of Symfony Web Debug Toolbar -->
 <div id="sfMiniToolbar-{{ token }}" class="sf-minitoolbar" data-no-turbolink>
-    <a href="#" title="Show Symfony toolbar" tabindex="-1" id="sfToolbarMiniToggler-{{ token }}" accesskey="D">
+    <button class="show-botton" type="button" title="Show Symfony toolbar" id="sfToolbarMiniToggler-{{ token }}" accesskey="D" aria-expanded="false" aria-controls="sfToolbarMainContent-{{ token }}">
```

Comment:
Maybe did you mean `show-button` to `show-botton`?

Reply:
That class is not really needed. I've just removed it.